Tests/Unit/Controller/Application/AbstractApplicationControllerTest.php

```diff
@@ -170,6 +171,13 @@ public function setsTypeConverterOptionForDateTime()
         $arguments->getArgument("application")->willReturn($argument->reveal());
         $configuration->forProperty('birthday')->willReturn($configuration->reveal());
 
+
+        $configuration->forProperty('files.999')->shouldBeCalled()->willReturn($configuration->reveal());
```

Comment:
You can drop the `shouldBeCalled()` here, it's not something we really care about.